examples/minimal.rs

```diff
@@ -1,9 +1,8 @@
-use ::auto_decode::auto_decode;
+use handler_macro::event_handler;
```

Comment:
Is there a reason this is a separate package?

I like the name of the macro but probably would expose it from the `radix_event_stream` package.
You could have something like `radix_event_stream::macros::event_handler`. If there is something against it you might also want to expose the important things at the top level like

`radix_event_stream::event_handler`

Reply:
There is a technical reason for this:

[rust book](https://doc.rust-lang.org/book/ch19-06-macros.html#procedural-macros-for-generating-code-from-attributes)

_"When creating procedural macros, the definitions must reside in their own crate with a special crate type. This is for complex technical reasons that we hope to eliminate in the future."_

Regarding `radix_event_stream::event_handler`, we already have an `event_handler` module so I would indeed do `radix_event_stream::macros::event_handler`, which is nicer, yes.